modules/tensor_mechanics/test/tests/scalar_material_damage/scalar_material_damage.i

```diff
@@ -0,0 +1,175 @@
+# This is a basic test of the system for continuum damage mechanics
+# materials. It uses ScalarMaterialDamage for the damage model,
+# which simply gets its damage index from another material. In this
+# case, we prescribe the evolution of the damage index using a
+# function. A single element has a fixed prescribed displacement
+# on one side that puts the element in tension, and then the
+# damage index evolves from 0 to 1 over time, and this verifies
+# that the stress correspondingly drops to 0.
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 3
+  nx = 1
+  ny = 1
+  nz = 1
+  elem_type = HEX8
+  displacements = 'disp_x disp_y disp_z'
+[]
+
+[Variables]
+  [disp_x]
+    order = FIRST
+    family = LAGRANGE
+  []
+  [disp_y]
+    order = FIRST
+    family = LAGRANGE
+  []
+  [disp_z]
+    order = FIRST
+    family = LAGRANGE
+  []
+[]
+
+[AuxVariables]
+  [stress_xx]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [strain_xx]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [damage_index]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+[]
+
+[Kernels]
+  [TensorMechanics]
+    displacements = 'disp_x disp_y disp_z'
```

Comment:
this is the legacy action. Switching to the master action would allow the use of `generate_outputs`, which would shorten this input considerably